data/stk_config.xml

```diff
@@ -122,6 +122,13 @@
   <replay max-time="600" delta-t="0.05"  delta-pos="0.1"
           delta-angle="0.5" />
 
+  <!--  Determines the minimap related values. 
+        size: The size of the minimap (scaled afterwards) 480 = full screen height)
```

Comment:
480 = full screen height? Not sure I get that part

Reply:
Look up the code in race_gui.cpp to understand why 480. This is not something I've touched in this PR.

I've put this information in the comment to guide (advanced) users who'd like to tweak their settings ; though ideally their should be a deeper rewrite of race_gui.cpp.